front-end/src/Views/MyCVsPage/MyCVsPage.jsx

```diff
@@ -27,17 +29,38 @@ class MyCVsPage extends React.Component {
                 }));
     };
 
+    cutItem = async (cvId) => {
+        let deleted;
+        try {
+            deleted = await deleteCV(this.context.token, cvId);
+        } catch(e) {
+            console.log(e);
+            this.setState({
+                delError: true
+            });
+        }
+        if (deleted) {
+            let cvList = this.state.cvs;
+            const indexToCut = cvList.findIndex(cv => cv.cv_id = cvId);
+            cvList.splice(indexToCut, 1);
+            this.setState({
+                cvs: cvList
+            });
```

Comment:
`cvList` to nie jest kopia `this.state.cvs`, tylko inna zmienna wskazująca na tą samą tablicę, więc ten fragment modyfikuje state bez użycia setState, a później ustawia jako cvs tą samą tablicę. Zamiast tego można np użyć funkcji filter która zwraca kopię tablicy:
```suggestion
            this.setState({
                cvs: this.state.cvs.filter(cv => cv.cv_id !== cvId)
            });
```